docs/api/queries/get.mdx

```diff
@@ -143,6 +143,12 @@ cy.get('form').within(() => {
 })
 ```
 
+#### Find an anchor element by its label/text, remove some attribute and click it
+
+```javascript
+cy.get('a', { name: 'My link text' }).invoke('removeAttr', 'target').click();
```

Comment:
The syntax here is not correct - it looks like a mix of Cypress and Testing Library syntax, the following would work though:

```suggestion
cy.contains('a', 'hello').invoke('removeAttr', 'target').click();
```

Reply:
@marktnoonan 

If the example is changed from `cy.get()` to `cy.contains()`, would it then need to be added to https://github.com/cypress-io/cypress-documentation/blob/main/docs/api/queries/contains.mdx instead?